src/renderer/components/inputs/InputContainer.tsx

```diff
@@ -123,7 +123,7 @@ export const HandleWrapper = memo(
                             position: 'absolute',
                             top: '50%',
                             left: '50%',
-                            height: '30px',
+                            height: '45px',
```

Comment:
What's this about?

Reply:
I had been working on different change and forgot to revert this

Reply:
Actually I didn't forget to revert this, this is actually consistent with the other handle now https://github.com/chaiNNer-org/chaiNNer/blob/12e219c2fa504fe0bda7c7d4926e127ca85b524e/src/renderer/components/outputs/OutputContainer.tsx#L107

Reply:
gonna resolve this because I think of it as a bugfix